### DISCUSSION TITLE
Jira webhook in GitHub repository

### DISCUSSION THREAD

**Discussion by berserker5000** [community/community#68197] [New to GitHub] [answered]
Hi all. I'm facing a problem with webhooks for Jira. Can somebody tell me, how can I create/change webhooks for a new Jira?
The situation is next:
We had an on-prem jira with the URL `jira.example.com`, we moved to the cloud jira and now we have the URL `company.example-cloud.com/jira` and in our webhooks, we do have `https://jira.example.com/rest/bitbucket/1.0/webhook/github` but when I'm changing it to the `https://company.example-cloud.com/jira/rest/bitbucket/1.0/webhook/github` it doesn't work. Any suggestions on what or where I can check? Strange thing that in Jira we can see GitHub progress (branch, pr, commit) but in GitHub repo during PR check commit is pointing to the old Jira ticket/URL. I think it is because of webhooks, but maybe not. 
Please advice.

**Comment by queenofcorgis** [comment]
Hi there - would you mind sharing where it isn't working in GitHub or Jira?

**Reply by berserker5000** [reply]
Hi @queenofcorgis, it is not working properly in GitHub. So it is showing us the wrong URL to JIRA.

**Reply by ghostinhershell** [reply]
Hey @berserker5000, thanks for confirming that. For added context, are you currently using the [Jira Software + GitHub app](https://github.com/marketplace/jira-software-github), or another app to create that integration between Jira and GitHub?

**Reply by berserker5000** [reply, marked as answer]
Dear @ghostinhershell , thanks for your assistance, I've found the problem. So basically I had no idea about Autolink, and that is where the problem was. I did change all the links and now everything works as expected.

**Reply by ghostinhershell** [reply]
So happy to hear it @berserker5000! 🚀

**Comment by queenofcorgis** [comment]
We’re glad you could get this solved and we want to make sure others can do the same so we’ve reopened this discussion for better discoverability for our members.
